README.md

```diff
@@ -43,24 +43,35 @@ npm install superjson
 Add the plugin to your `.babelrc`.
 If you don't have one, create it.
 
-
 ```json5
 {
-  "presets": ["next/babel"],
-  "plugins": [
-    ...
-    "superjson-next" // 👈
-  ]
+  presets: ['next/babel'],
+  plugins: [
+    ...'superjson-next', // 👈
```

Comment:
```suggestion
    ...
    'superjson-next', // 👈
```

Reply:
Because of prettier. Need to add a prettier ignore block around this

Reply:
0d4fc28